plugins/kubernetes/db/migrate/20150827181056_create_kubernetes_cluster.rb

```diff
@@ -0,0 +1,11 @@
+class CreateKubernetesCluster < ActiveRecord::Migration
+  def change
+    create_table :kubernetes_clusters do |t|
+      t.string :name
+      t.string :description
+      t.string :config_filepath
```

Comment:
Currently, the logic is expecting a filepath on the Samson server.  Yeah, I know this isn't ideal.

The config contains a bunch of information, including stuff like base-64 encoded certificates, CAs, etc. Creating a UI to manage all that would be a pain.  The config file is automatically generated by the Kubernetes installation scripts when you run the Vagrant installation, and could be generated by a config management system or scripts.

I'd like to start with this, and possibly add a future enhancement to allow inputting config information by a UI.